integration_tests/pkg/snap/sessions/docker_test.go

```diff
@@ -85,10 +85,13 @@ func TestSnapDockerSession(t *testing.T) {
 				tags,
 			)
 			So(err, ShouldBeNil)
-			So(dockerHandle.IsRunning(), ShouldBeTrue)
-			dockerHandle.Wait()
-			time.Sleep(5 * time.Second) // One hit does not always yield results.
-			dockerHandle.Stop()
+			defer dockerHandle.Stop()
+
+			So(dockerHandle.Status(), ShouldEqual, executor.RUNNING)
+			time.Sleep(10 * time.Second)
```

Comment:
@skonefal - as a developer I would look at the test as example use case. It should not instruct user to run `Stop()` after `Wait(0)`.